packages/documentation/src/stories/components/back-to-top/back-to-top.docs.mdx

```diff
@@ -0,0 +1,33 @@
+import { Canvas, Controls, Meta, Source } from '@storybook/blocks';
+import * as BackToTopStories from './back-to-top.stories.ts';
+import StylesPackageImport from '@/shared/styles-package-import.mdx';
+import BackToTop from './back-to-top.sample.html?raw';
+
+<Meta of={BackToTopStories} />
+
+<div className="docs-title">
+ # Back To Top
+
+  <nav>
+    <link-design of={JSON.stringify(BackToTopStories)}></link-design>
+  </nav>
+</div>
+
+<p className="lead">
+  Add this component to display a Back to Top button that appears when the user scrolls past a
+  specified threshold, allowing for smooth scrolling back to the top of the page.
```

Comment:
It is a good practice to re-use the introduction text from Figma in the docs: https://www.figma.com/design/JIT5AdGYqv6bDRpfBPV8XR/Foundations-%26-Components-Next-Level?node-id=517-3594&node-type=frame&t=LZ05qAgicQadYqKo-0